CMakeLists.txt

```diff
@@ -18,7 +18,7 @@
 cmake_minimum_required(VERSION 3.9)
 
 # Build the sdk targets
-project(olp-cpp-sdk VERSION 1.10.0)
+project(olp-cpp-sdk VERSION 1.11.0)
```

Comment:
Change Copyright year to `2019-2021`.
Please apply for all the below files also.